src/components/Loader/PersonLink.tsx

```diff
@@ -0,0 +1,68 @@
+import classNames from 'classnames';
+import { FC } from 'react';
+import { Link } from 'react-router-dom';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+  selectedPerson: string;
+};
+
+export const PersonLink: FC<Props> = ({
+  person: {
+    name,
+    sex,
+    born,
+    died,
+    fatherName,
+    motherName,
+    slug,
+    mother,
+    father,
+  },
+  selectedPerson,
+}) => {
+  const isSelected = (value: string) => value === selectedPerson;
+
+  const checkParent = (
+    parent: Person | undefined,
+    parentName: string | null,
```

Comment:
Just switch order for the arguments and don't pass parent if you don't need to
```suggestion
    parentName: string | null,
    parent?: Person,
```

Reply:
fixed